Content.Server/Fax/FaxSystem.cs

```diff
@@ -350,6 +352,18 @@ private void OnDestinationSelected(EntityUid uid, FaxMachineComponent component,
         SetDestination(uid, args.Address, component);
     }
 
+    // imp start
+    private void OnTriggerPortFired(EntityUid uid, FaxMachineComponent component, SignalReceivedEvent args)
+    {
+        // need to make a fake FaxSendMessage to pass into Send()
+        // normally the client would make this message when a player sends it,
+        // but since this is being triggered by a remote signaler (handled on server)
+        // we need to make one on the server instead to pass it into Send()
+        FaxSendMessage fakeSendMessage = new FaxSendMessage();
+        Send(uid, component, fakeSendMessage);
+    }
+    // imp end
+
```

Comment:
```suggestion
    // imp start
    private void OnTriggerPortFired(Entity<FaxMachineComponent> ent, ref SignalReceivedEvent args)
    {
        Send(ent, ent.Comp, new FaxSendMessage();
    }
    // imp end

```

using `Entity<T>` rather than seperate ent/comp fields is better practice generally
what params does the usual `FaxSendMessage` have? is there anything we need to pass into it?